iptables-wrapper-installer.sh

```diff
@@ -109,12 +109,22 @@ set -eu
 # kubelet) has already created some iptables rules.
 EOF
 
+cat >> "${sbin}/iptables-wrapper" <<EOF
+num_legacy_lines=\$( (iptables-legacy-save || true; ip6tables-legacy-save || true) 2>/dev/null | grep '^-' | wc -l)
+chains_created_by_kubelet=":KUBE-MARK-DROP|:KUBE-MARK-MASQ|:KUBE-POSTROUTING"
+kubelet_legacy_chains=\$( (iptables-legacy-save || true; ip6tables-legacy-save || true) 2>/dev/null | grep -E \${chains_created_by_kubelet} | wc -l)
```

Comment:
If we're going to do this, it seems like it would be better to only make a single call and save the output to a file.
(If this gets run when there are already a lot of iptables rules in use, then just running `iptables-save` can be quite slow.)

Reply:
good point. Fixed in my second commit